desktop/src/preferences/read.rs

```diff
@@ -51,6 +51,10 @@ pub fn read_preferences(input: &str) -> ParseDetails<SavedGlobalPreferences> {
         result.mute = value;
     };
 
+    if let Some(value) = document.get_bool(&mut cx, "enable_openh264") {
+        result.enable_openh264 = value;
+    };
```

Comment:
Can we get a trivial read/write test for this, so we don't accidentally break this in the future?

Reply:
Honestly, I'm not really seeing how _this_, of all things, could be broken "accidentally", but sure, more tests are more good! So I mostly just copied the test for `mute` - another boolean preference -, and substituted `enable_openh264`.

Reply:
Sorry for being so nit-picky :sweat_smile:, but we also have write tests in `preferences/write.rs` (mostly for round trip purposes), so if you could add one for this it would be great!

Reply:
No problem, please do be! I'll get right on it, just haven't noticed them myself.

Reply:
Done.